src/main/java/org/opensearch/ad/AnomalyDetectorPlugin.java

```diff
@@ -243,7 +209,7 @@ public Collection<Object> createComponents(
             AnomalyDetectorSettings.MAX_UPDATE_RETRY_TIMES
         );
         this.clusterService = clusterService;
-
+    
```

Comment:
This whitespace look suspicious. Spotless is ok with it?

Reply:
@joshpalis do you want to remove the whitespaces from the=is file?